proxy_agent/src/proxy.rs

```diff
@@ -103,8 +103,13 @@ fn get_process_info(process_id: u32) -> (PathBuf, String) {
     let mut process_cmd_line = UNDEFINED.to_string();
 
     let pid = Pid::from_u32(process_id);
-    let mut sys = System::new();
-    sys.refresh_processes();
+    let sys = System::new_with_specifics(
+        RefreshKind::new().with_processes(
+            ProcessRefreshKind::new()
+                .with_cmd(UpdateKind::Always)
+                .with_exe(UpdateKind::Always),
+        ),
+    );
     if let Some(p) = sys.process(pid) {
         process_name = p.exe().to_path_buf();
```

Comment:
```
        process_name = match p.exe() {
            Some(path) => path.to_path_buf(),
            None => PathBuf::default(),
        };
```

Reply:
Sorry about that. The code update was incomplete. Fixed now.